migrations/20241114200715_addScanDate.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `books` ADD `scanDate` text;
```

Comment:
🤔 should this be an integer timestamp, or is it better as an ISO string (as-is)? I'm not sure; and curious about your thoughts in terms of the tradeoff.

Reply:
As discussed, keeping the type as is. Some further investigation might be warranted about drizzle/sqlite/datetime best practices, but non-blocking.

Reply:
If I'm reading the drizzle docs correctly; Drizzle has a built-in `mode` for `integer` fields -- but not for `text` fields to automagically convert to dates:

https://orm.drizzle.team/docs/guides/timestamp-default-value#sqlite

A [pull request](https://github.com/drizzle-team/drizzle-orm/pull/2195) exists for adding something similar for text / date-time fields; but it hasn't landed yet. That's pushing me towards using a unix timestamp here after all.